packages/@sanity/default-login/src/LoginWrapper.js

```diff
@@ -41,18 +44,37 @@ export default class LoginWrapper extends React.PureComponent {
   state = {isLoading: true, user: null, error: null}
 
   userSubscription = null
+  tokenSubscription = null
 
   constructor(props) {
     super(props)
 
     let sync = true
+    this.tokenSubscription = ENABLE_CROSS_WINDOW_TOKEN_LOGIN_SYNC
+      ? authToken$.subscribe((token) => {
+          if (token) {
+            if (!this.state.user) {
+              this.handleRetry()
+            }
+            return
+          }
+          if (sync) {
+            // eslint-disable-next-line react/no-direct-mutation-state
+            this.state = {...this.state, ...{user: null}}
```

Comment:
is `{...this.state, ...{user: null}}` different from `{...this.state, user: null}`?

Reply:
Nope!